faqs.html

```diff
@@ -225,11 +224,21 @@
 											</div>
 										</div> -->
 										<div class="tg-panel">
-											<h4><span>Q.</span>I need to cancel my ticket due to unavoidable cicumstances. What is the refund policy?</h4>
+											<h4><span>Q.</span>I need to cancel my ticket due to unavoidable circumstances. What is the refund/transfer policy?</h4>
 											<div class="tg-panelcontent">
 												<div class="tg-description">
-													<p><span>A.</span>We allow tickets to be cancelled, but cancellation might be barred in the last few days leading up to the conference. This space will be updated if cancellation is barred at any time.
-													</p>
+												<p><span>A.</span>- All tickets are transferable.</p>
+												<p class="tg-offset">- Early bird tickets cannot be canceled. They can only be transferred.</p>
+												<p class="tg-offset">- All other tickets can be canceled till 10th Oct 2017.</p>
+												<p class="tg-offset">- Tickets can be transferred till 10th Oct 2017 using the ticket transfer <a href="https://www.explara.com/manage-tickets">link</a>.</p>
+												</div>
+											</div>
+										</div>
+										<div class="tg-panel">
+											<h4><span>Q.</span>Do I have to buy a seperate ticket for the Dev Sprint?</h4>
+											<div class="tg-panelcontent">
+												<div class="tg-description">
+													<p><span>A.</span>No, you don't have to buy a seperate ticket for Dev Sprint. But you'll need to have either a Conference Ticket or a Workshop Ticket to be able to enter Dev Sprint.</a></p>
```

Comment:
Also, add here that due to space constraint, entry to Dev Sprints will be on first-come first-serve basis